rts/Sim/Units/Scripts/CobEngine.cpp

```diff
@@ -31,7 +31,7 @@ CR_REG_METADATA(CCobEngine::SleepingThread, (
 	CR_MEMBER(wt)
 ))
 
-static const char* const numCobThreadsPlot = "CobThreads";
+// static const char* const numCobThreadsPlot = "CobThreads";
```

Comment:
This seems to be used by Tracy calls below. Perhaps the compiler complained it looks unused because you were compiling without Tracy?